src/Perspex.SceneGraph/Animation/CrossFade.cs

```diff
@@ -8,12 +8,29 @@
 
 namespace Perspex.Animation
 {
+    // TODO: Perhaps we want a common base for Transitions with Duration so that we can 
+    // consolidate the property declarations, etc
+    //
+
     /// <summary>
     /// Defines a cross-fade animation between two <see cref="IVisual"/>s.
     /// </summary>
-    public class CrossFade : IPageTransition
+    public class CrossFade : PerspexObject, IPageTransition
```

Comment:
Ah good point. Humm, it might be useful down the road to support Binding the duration, not sure. What's your general belief on making everything bindable or not? Is there a downside?